# departamentos/domain/exceptions/departamento_exception.py

class DepartamentoError(Exception):
    pass
